plonky3/src/stark.rs

```diff
@@ -126,11 +126,11 @@ impl<T: FieldElement> Plonky3Prover<T> {
         }
 
         // get the config
-        let config = get_config();
+        let config = get_config_goldilocks();
```

Comment:
Since we are now making this generic over the field `F`, this cannot refer to a particular field (goldilocks) but rather should call a method of the `FieldElementMap` trait. Why suggests moving `get_config` to the trait.